package.json

```diff
@@ -603,14 +631,14 @@
         "sass-loader": "^12.1.0",
         "style-loader": "^3.2.1",
         "ts-loader": "^9.2.6",
-        "typescript": "^3.9.9",
+        "typescript": "^4.7.0",
         "vscode-test": "^1.5.2",
         "webpack": "^5.73.0",
         "webpack-cli": "^4.10.0"
     },
     "dependencies": {
         "@popperjs/core": "^2.10.1",
-        "@spcl/sdfv": "^1.0.0",
+        "@spcl/sdfv": "^1.0.29-beta.12",
```

Comment:
prerelease...

Reply:
Yes the package is pre-release, but this is only for as long as the extension itself is in pre-release. This is necessary to have the localview integration, since that has not made it into the main release of SDFV yet. After a period of about two weeks, where the feature is showcased in the extension pre-release, the proper `v1.0.29` of SDFV will be released at the same time as this PR is taken from pre-release (`v1.3.1`) to release (`v1.2.1`).

Man I hope VSCode soon supports semver tagging, this 1.3.x for pre-release, 1.2.x for release is confusing..